WDAC-Policy-Wizard/app/src/ConfigTemplate_Control.cs

```diff
@@ -623,10 +572,78 @@ private bool ReadSetRules(object sender, EventArgs e)
                 this._MainWindow.Policy.TemplatePath = xmlPathToRead; 
             }
             
-
             return true; 
         }
 
+        /// <summary>
+        /// Looks up the policy xml path to parse the rule-options
+        /// </summary>
+        /// <returns></returns>
+        private string GetTemplatePath()
+        {
+            string xmlPathToRead = String.Empty;
+
+            // If we are editing a policy, read the EditPolicyPath
+            // We need to know whether we are editing a base or supplemental policy
+            if (this.Policy.PolicyWorkflow == WDAC_Policy.Workflow.Edit)
+            {
+                xmlPathToRead = this._MainWindow.Policy.EditPolicyPath;
+            }
+
+            // If we are supplementing a policy, we need to mirror the rule options of the base so they do not conflict
+            else if (this.Policy._PolicyType == WDAC_Policy.PolicyType.SupplementalPolicy)
+            {
+                xmlPathToRead = this._MainWindow.Policy.BaseToSupplementPath;
+            }
+            else
+            {
+                if (this.Policy._Format == WDAC_Policy.Format.MultiPolicy)
+                {
+                    // Multi-policy Format Policy Templates
+                    switch (this.Policy._PolicyTemplate)
+                    {
+                        case WDAC_Policy.NewPolicyTemplate.WindowsWorks:
+                            // Windows Works Mode 
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, Properties.Resources.WindowsTemplate);// "DefaultWindows_Audit.xml");
```

Comment:
Now not necessary

Reply:
This is still necessary. Maybe I am missing something?

Reply:
I was (and all these not necessary comments are) referring to the comment at the end.

Reply:
Ah, got it. Just clean up the comment?

Reply:
yes